include/ignition/gazebo/Types.hh

```diff
@@ -78,6 +78,8 @@ namespace ignition
     /// \brief A unique identifier for a component instance. The uniqueness
     /// of a ComponentId is scoped to the component's type.
     /// \sa ComponentKey.
+    // TODO(anyone) remove this in ign-gazebo7. This is no longer used in
+    // ign-gazebo6
     using ComponentId = int;
```

Comment:
I left a TODO note to remove this because I don't think it's possible to mark a type alias as deprecated, right?